translations/base-nl.yaml

```diff
@@ -197,21 +198,20 @@ dialogs:
         desc: Je kunt het je niet veroorloven om de selectie te plakken! Weet je zeker
             dat je het wil knippen?
     editSignal:
-        title: Set Signal
+        title: Stel het signaal in.
         descItems: "Kies een ingesteld item:"
         descShortKey: ... of voer de <strong>short key</strong> van een vorm (Die je
```

Comment:
You left "short key" untranslated here but translated it as "sleutel" [in the shape viewer](https://github.com/MisterPengu/shapez.io/blob/f8df08bfaf3e07a5773d45998f3c8cca32a304de/translations/base-nl.yaml#L347), should pick one and be consistent